draft-ietf-httpbis-client-cert-field.md

```diff
@@ -87,7 +87,7 @@ specific details from the certificate needed also vary with the application
 requirements. In order for these types of application deployments to work in
 practice, the reverse proxy needs to convey information about the client
 certificate to the origin application server. A common way this information is
-conveyed in practice today is by using non-standard fields to carry the
+conveyed in practice at the time of writing is by using non-standard fields to carry the
```

Comment:
I like this. I might restructure the sentence to start with the statement e.g. "at the time of writing, a common way to convey this information is to use non-atandard fields..."

Reply:
I almost did exactly that but opted for a more narrow change. I'm happy to take the suggestion though. 

Overall, I was hopeful that this much smaller change would still alleviate (at least some of) your concern about the text reading odd in the future.

Reply:
[095b564](https://github.com/httpwg/http-extensions/pull/2307/commits/095b564a712df2801765b8c04a6317f285782b7b) does that restructure